lib/task/import/csvImportTask.class.php

```diff
@@ -1007,7 +1066,7 @@ public function execute($arguments = array(), $options = array())
             $this->log($e->getMessage(), sfLogger::ERR);
```

Comment:
Those code blocks are in different lambda functions, and I don't think that I can set a local var and have it be avail between the two.   The first lambda is conditionally called and if a var is set in there for the next function, it will be a time bomb I think.